src/adk/api/remote_api.py

```diff
@@ -724,6 +724,41 @@ def experiments_list(self) -> List[ExperimentType]:
 
         return experiment_list
 
+    def __get_backend_type(self, backend_type: str) -> List[BackendTypeType]:
+        """
+        Get the backend info for the remote backend in the experiment data.
+
+        Returns:
+            0 or 1 backend types with this type
+        """
+        backend_types = self.__qne_client.list_backendtypes()
+        # "type" is required
+        backends = [backend for backend in backend_types if str(backend["name"]).lower() == backend_type.lower()]
```

Comment:
maybe rename backend to backend_type, since you are looping over backend_types?

Reply:
The name backend_type would clash with the parameter backend_type.
I rename the parameter too